concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-306	306	It (feminine) was	Elle a été	statu habūtu	102		
+Gauchat-1925-480-307	307	badly sung	mal chantée	male cantata	102		
+Gauchat-1925-480-308	308	they were (badly) sung	elles ont été (mal) chantées	cantatas	102		
+Gauchat-1925-480-309	309	(They) should	(Ils) devraient	debēre + (hab)ēbant	104		
+Gauchat-1925-480-310	310	all	tous	tōttī	104	98	ALL
+Gauchat-1925-480-311	311	shut up	se taire	qu(i)etiare	104	48	BE SILENT
+Gauchat-1925-480-312	312	All (feminine)	Toutes	tōttas	104		
+Gauchat-1925-480-313	313	these women	ces femmes	fēminas	104		
+Gauchat-1925-480-314	314	they do not know	ne savent pas	sapunt	104		
+Gauchat-1925-480-315	315	to sing	chanter	cantare	104	1261	SING
+Gauchat-1925-480-316	316	One cannot	On ne peut pas	pŏtet	106		
+Gauchat-1925-480-317	317	know everything	tout savoir	sapēre	106		
+Gauchat-1925-480-318	318	That does not prove	Cela ne prouve	prŏbat	106		
+Gauchat-1925-480-319	319	nothing	rien	rĕm	106		NOTHING
+Gauchat-1925-480-320	320	One keeps from	On défend	defēndit	106		
+Gauchat-1925-480-321	321	the children	aux enfants	infantes	106	2099	CHILD
+Gauchat-1925-480-322	322	(to) swear	(de) jurer	jurare	108	1939	SWEAR
+Gauchat-1925-480-323	323	(to) cry	(de) pleurer	plorare	108	1839	CRY
+Gauchat-1925-480-324	324	(to) jump	(de) sauter	saltare	108	1829	JUMP
+Gauchat-1925-480-325	325	This little girl	Cette fillette	filia + ĭtta	108		
+Gauchat-1925-480-326	326	has made	a fait	factu	108		
+Gauchat-1925-480-327	327	her prayer	sa prière	precaria	108		
+Gauchat-1925-480-328	328	She is sleepy	Elle a sommeil	sŏmnu	110		
+Gauchat-1925-480-329	329	She sleeps	Elle dort	dŏrmit	110		
+Gauchat-1925-480-330	330	already	déjà	jam	110	2576	ALREADY
+Gauchat-1925-480-331	331	when	quand	quando	110	1238	WHEN
+Gauchat-1925-480-332	332	you come	vous venez	venītis	110		
+Gauchat-1925-480-333	333	to go to bed	vous coucher	collocare	110	215	LIE DOWN
+Gauchat-1925-480-334	334	My brother	Mon frère	fratre	112	1262	BROTHER
+Gauchat-1925-480-335	335	is bored	s’ennuie	inŏdiat	112		
+Gauchat-1925-480-336	336	alone	tout seul	solĭttu	112	1964	ALONE
+Gauchat-1925-480-337	337	you would like	vous voudriez	volēre + (hab)ebatis	112		
+Gauchat-1925-480-338	338	that I do not	que je ne le	ĭllu	112		
+Gauchat-1925-480-339	339	know (not a bit)	sache pas	sapiam	112		
+Gauchat-1925-480-340	340	I cry	Je pleure	plōro	114		
+Gauchat-1925-480-341	341	slowly	doucement	-mĕnte	114	2165	SLOWLY
+Gauchat-1925-480-342	342	so that we don’t hear	pour qu’on n’entende pas	hŏmo audiat	114		
+Gauchat-1925-480-343	343	(I) can	(Je) sais	sa(p)io	114		
+Gauchat-1925-480-344	344	some	un peu	paucu	114	1241	SOME
+Gauchat-1925-480-345	345	read	lire	lĕgere	114	724	READ
+Gauchat-1925-480-346	346	I say that to nobody	Je ne le dis	dīco	116		
+Gauchat-1925-480-347	347	than to them	qu’à eux	illōru	116		
+Gauchat-1925-480-348	348	It beats	Il frappe	fĕrit	116		
+Gauchat-1925-480-349	349	one o'clock	une heure	hōra	116	1353	HOUR
```

Comment:
That's the time, not the noun, I think.

Reply:
I think you're right here, I'll unmap it.

Reply:
"une heure" is in french the same as "l'heure", so if you isolate it, you have "hour", but if we stick to the strategy of mapping only the most clear-cut cases, we can unmap it. It would be mapped as "hour" in an updated list in which all words are isolated from the phrases.